quirks/password-rules.json

```diff
@@ -239,6 +239,9 @@
     "google.com": {
         "password-rules": "minlength: 8; allowed: lower, upper, digit, [-!\"#$%&'()*+,./:;<=>?@[^_{|}~]];"
     },
+    "gwl.greatwestlife.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!#$%-_=+<>];"
```

Comment:
`-` needs to be the first character of the class.

Reply:
Good catch!